std_execution.bs

```diff
@@ -292,13 +292,33 @@ essential when writing user code using standard execution concepts; we have also
 
 ## Prior art ## {#intro-prior-art}
 
-This proposal builds upon and learns from years of prior art with asynchronous and parallel programming frameworks in C++.
+This proposal builds upon and learns from years of prior art with asynchronous and parallel programming frameworks in C++. In this section, we discuss async abstractions that have previously been suggested as a possible basis for asynchronous algorithms and why they fall short.
+
+### Futures ### {#intro-prior-art-futures}
+
+A future is a handle to work that has already been scheduled for execution. It is the receiving end of a communication channel. The other end is a promise, used by the concurrent operation to set the result.
```

Comment:
I wonder if it's possible to reword this to not use the word "receiving", because here it has a meaning that's a reverse of the meaning in "receiver".